Modules/ContentKit/Sources/Content/_NewSystem/CurationType.swift

```diff
@@ -0,0 +1,70 @@
+// Leka - iOS Monorepo
+// Copyright APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+import Combine
+import LocalizationKit
+import SwiftUI
+
+// MARK: - CurationType
+
+public enum CurationType: String, Codable {
```

Comment:
we might have talked about it but i can't remember what we said: why do you need a curation type? why not add `.curation` to `ContentType`
we already have the resources inside, so we might as well put it here instead of duplicating.